docs/core/config/auth/databases/ldap.md

```diff
@@ -254,7 +269,7 @@ This should report "Verify return code: 0 (ok)".
 It's possible to use SASL binds instead of the regular simple binds if your
 LDAP library supports them.
 
-See the `sasl_*` settings in `dovecot-ldap-example.conf`.
+See the `ldap_sasl_*` settings in `dovecot-ldap-example.conf`.
```

Comment:
ldap_auth_sasl_* settings. Also can't refer to the config file. Should link to the ldap_auth_sasl_mechanism setting directly.